service/src/test/java/org/mskcc/smile/service/TempoServiceTest.java

```diff
@@ -103,8 +104,16 @@ public void initializeMockDatabase() throws Exception {
         // mock request id: MOCKREQUEST1_B
         MockJsonTestData request1Data = mockDataUtils.mockedRequestJsonDataMap
                 .get("mockIncomingRequest1JsonDataWith2T2N");
-        SmileRequest request1 = RequestDataFactory.buildNewLimsRequestFromJson(request1Data.getJsonString());
+        SmileRequest request1 =
+                RequestDataFactory.buildNewLimsRequestFromJson(request1Data.getJsonString());
         requestService.saveRequest(request1);
+
+        // mock request id: 22022_BZ
+        MockJsonTestData request2bData = mockDataUtils.mockedRequestJsonDataMap
```

Comment:
mock test samples from this request are referenced in the new unit test added for handling cohort updates